code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -4,6 +4,8 @@
 #define JUMP_COOLDOWN 2 SECONDS
 ///Time between being dropped and going idle
 #define ACTIVATE_TIME 4 SECONDS
+///Time to go active after impacting a carbon on a toss
+#define IMPACT_TIME 1.5 SECONDS
```

Comment:
these should be vars on the facehugger instead of a global define, easier to balance each one on each point and doens't need a modifier that makes knowing the value harder